src/torchpme/calculators/calculator_dipole.py

```diff
@@ -0,0 +1,161 @@
+from typing import Optional, Union
+
+import torch
+from torch import profiler
+
+from .._utils import _validate_parameters
+from ..lib import generate_kvectors_for_ewald
+from ..potentials import PotentialDipole
+
+
+class CalculatorDipole(torch.nn.Module):
+    """TODO: Add docstring"""
```

Comment:
You mind adding docstrings here?

I can prepare and push the rst entries to your branch.